lib/features/children/generosity_challenge/models/day.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:equatable/equatable.dart';
+
+class Day extends Equatable {
+  const Day({
+    required this.dateCompleted,
+  });
```

Comment:
I imagine that we will expand the Day to also have more challenge data like day title, and daily task/message. What do you think?